virtual_rainforest/core/base_model.py

```diff
@@ -1,4 +1,4 @@
-"""The :mod:`~virtual_rainforest.core.model` module provides a consistent API across
+"""The :mod:`~virtual_rainforest.core.base_model`module provides a consistent API across
```

Comment:
needs to be a space between base_model` and module here, otherwise the docstring doesn't render properly